src/web/src/components/Posts/PostAvatar.tsx

```diff
@@ -26,6 +26,11 @@ const useStyles = makeStyles((theme: Theme) =>
         width: '2.5em',
         height: '2.5em',
       },
+      [theme.breakpoints.down(375)]: {
+        fontSize: '1.7em',
+        width: '2.5em',
+        height: '2.5em',
```

Comment:
Do we need to restate `width` and `height`? If you removed it, I think it will inherit from the previous breakpoint.

Reply:
Yeah that's what I noticed too, I think it's ok to remove it